tests/testthat/_snaps/utils.md

```diff
@@ -0,0 +1,21 @@
+# fail_on_missing_package
+
+    Code
+      fail_on_missing_package("logger", "9.9.9", call = quote(f()))
+    Condition
+      Error:
+      ! Please install min. 9.9.9 version of logger to use f
+    Code
+      fail_on_missing_package("an.R.package-that-doesNotExists", call = quote(f()))
+    Condition
+      Error:
+      ! Please install the 'an.R.package-that-doesNotExists' package to use f
```

Comment:
```suggestion
      ! Please install the "an.R.package-that-doesNotExists" package to use f
```

Reply:
Hmmm this is because of (mis)use of `shQuote()` which I thought I fixed elsewhere.